recipes-kernel/linux/linux-nilrt-debug_git.bb

```diff
@@ -1,5 +1,5 @@
 DESCRIPTION = "NILRT linux kernel debug build"
-NI_RELEASE_VERSION = "master"
+NI_RELEASE_VERSION = "23.0"
 LINUX_VERSION = "5.10"
 LINUX_VERSION_xilinx-zynq = "4.14"
 LINUX_VERSION_qemu-zynq7 = "4.14"
```

Comment:
Oh; that's right. We were intending to keep the academic distro using the 4.14 master branch, but with a stapled commit - since we don't expect to add commits to that mainline.

If we're still OK with that plan, I can just drop this PR.

Reply:
I'll leave it to @gratian. Did you intend that with the sumo release branch as well?

Reply:
No; just academic.

Reply:
The academic 4.14 kernel branch: https://github.com/ni/linux/commits/nilrt-academic/master/4.14 does have a couple of extra commits added recently.

Not sure if we want to leave recipes pointing to master and update the hash or create a 23.0 kernel branch.

Reply:
We spoke about this offiline, but for web: the academic meta-nilrt branches are stapled to a kernel master ref commit which is current with the changes gratian linked above. So, there is no need to make a new kernel branch for it.